packages/editor/src/components/commands/index.js

```diff
@@ -261,10 +261,22 @@ function useEditorCommandLoader() {
 }
 
 function useEditedEntityContextualCommands() {
-	const { postType } = useSelect( ( select ) => {
-		const { getCurrentPostType } = select( editorStore );
+	const { postType, isViewable, status, link } = useSelect( ( select ) => {
+		const {
+			getCurrentPostType,
+			getEditedPostAttribute,
+			getEditedPostPreviewLink,
+		} = select( editorStore );
+		const { getPostType } = select( coreStore );
+		const isPublished = getEditedPostAttribute( 'status' ) === 'publish';
+
 		return {
 			postType: getCurrentPostType(),
+			isViewable: getPostType( getCurrentPostType() )?.viewable ?? false,
```

Comment:
```suggestion
		const _postType = getCurrentPostType();
		return {
			postType: _postType,
			isViewable: getPostType( _postType )?.viewable ?? false,
```

Reply:
Above suggestion depends on whether we need `isViewable` at all, and rather, test the `postType`